<launch>
  <include file="$(find tabletop_pushing)/launch/reactive_move.launch"/>
  <node pkg="tabletop_pushing" type="tabletop_push_node.py" name="tabletop_push_node" output="screen">
    <param name="collision_aware_ik" type="int" value="0"/>
    <param name="torso_z_offset" type="double" value="0.3"/>
    <param name="default_torso_height" type="double" value="0.26"/>
    <param name="look_point_x" type="double" value="0.7"/>

  </node>
</launch>
